cms/static/js/factories/sync_course.js

```diff
@@ -0,0 +1,145 @@
+define([
```

Comment:
can we keep this js file in edly too?

Reply:
This is the new written static js file. It's will not impact the openEdx core when moving forward with releases.